src/test/java/org/jabref/logic/shared/DBMSSynchronizerTest.java

```diff
@@ -124,8 +126,15 @@ public void testEntryRemovedEventListener() {
         assertEquals(bibEntry, actualEntries.get(0));
     }
 
-    @Test
-    public void testMetaDataChangedEventListener() {
+    @ParameterizedTest
+    @MethodSource("getTestingDatabaseSystems")
+    public void testMetaDataChangedEventListener(DBMSType dbmsType, DBMSConnection dbmsConnection, DBMSProcessor dbmsProcessor) throws Exception {
+
+        bibDatabase = new BibDatabase();
```

Comment:
The first three lines are shared between all tests, thus please extract them to the `setup` method.